### PR TITLE
updated documentation on the monaco protocol

### PR DESCRIPTION
**Pull request by druspencer**
- Bring details about the monaco protocol into the landing page
- Add a basic roadmap of upcoming work
- Give high level overview of what the protocol does
- Add information about markets available for development

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,3 +13,47 @@ Once you've done that, check out:
 - [Q&A](https://github.com/MonacoProtocol/sdk/discussions/categories/q-a)
 
 Any troubles, issues, questions, ideas you have - anything - raise them in discussions and the community will help you out.
+
+# Introduction
+
+The Monaco Protocol is a decentralized liquidity network for exchange-based applications built on the Solana blockchain. It serves as crucial infrastructure upon which third-party front-end applications for wagering on binary outcome events can be built while accessing a shared liquidity pool.
```
Comment (eoin-betdex) [inline README.md:19]: Given all the glossary discussions recently, we should make sure `wagering` is the right verb to use here.

Comment (eoin-betdex) [reply README.md:19]: (and throughout)

File: README.md [hunk 2]
```diff
@@ -13,3 +13,47 @@ Once you've done that, check out:
...
+
+## Why is the Monaco Protocol useful?
+
+- The Monaco Protocol streamlines the wagering process.
+- It helps provide greater liquidity for participants, reduce spreads and offer lower prices.
```
Comment (eoin-betdex) [inline README.md:24]: oxford comma? `, and`

Comment (druspencer) [reply README.md:24]: comma'd in oxford!

File: README.md [hunk 3]
```diff
@@ -13,3 +13,47 @@ Once you've done that, check out:
...
+## Is the Protocol audited?
+
+- The protocol is currently in the process of an audit.
+- Additional information on this will be released once the the audit is complete.
+
+## When Will the Protocol be on MainNet?
+
+- The protocol will launch on mainnet once auditing and testing is complete.
+- The protocol is currently deployed on the [devnet cluster](https://docs.solana.com/clusters#devnet) at [5Q2hKsxShaPxFqgVtQH3ErTkiBf8NGb99nmpaGw7FCrr](https://explorer.solana.com/address/5Q2hKsxShaPxFqgVtQH3ErTkiBf8NGb99nmpaGw7FCrr?cluster=devnet)
+- The authority of on devnet is [98CVwMftrhm6zutmV29frqRPfXsocbFnwjXVxYo7xbHX](https://explorer.solana.com/address/98CVwMftrhm6zutmV29frqRPfXsocbFnwjXVxYo7xbHX?cluster=devnet)
```
Comment (eoin-betdex) [inline README.md:43]: `of on`

Comment (druspencer) [reply README.md:43]: `off on`

File: README.md (hunk 1 above)
Comment (druspencer) [reply README.md:19]: yeah 😬

File: readme/the-roadmap.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+- Publish whitepaper
+- Form official foundation entity 
+
+# Technical
+
+- Complete audit of the protocol
+- Release admin client
+- Open source the monaco protocol
+- Complete DevNet testing prior to MainNet launch
```
[the remark this answers is not in the document]
Comment (druspencer) [reply readme/the-roadmap.md:15]: that double `nn` in mainnet... but you're right, should be consistent with the network

[the remark this answers is not in the document]
Comment (druspencer) [reply readme/the-roadmap.md:15]: should be consistent in these docs too I've used devnet elsewhere 🤦

File: README.md [hunk 5]
```diff
@@ -13,3 +13,47 @@ Once you've done that, check out:
...
+## How do I contribute to the Protocol?
+
+- More information about this will be available soon.
+- For now you can share ideas over on the [DevHub](https://github.com/MonacoProtocol/sdk/discussions).
+
+## Have any applications built on the Monaco Protocol?
+
+- BetDEX exchange is currently in closed beta.
+- You can learn more about the exchange on https://betdex.gitbook.io/betdex-beta/.
+
```
Comment (varunsudhakar) [inline README.md:56]: Can we add that community members from SAMO are a crypto-based prediction market?

File: readme/the-dev-environment.md [hunk 6]
```diff
@@ -13,11 +13,21 @@ Only interact with protocol addresses shipped with the client as these are verif
 
 ## Test Markets
 
-Currently there is no admin client to allow for developers to create their own markets. Until this client is shipped, markets will be regularly added to the protocol to enable development. Similarly, the token used to enter the market can be obtained via an airdrop whilst we work on a faucet.
+Currently there is no admin client to allow for developers to create their own markets. As such, market automation is in place to facilitate development:
+
```
Comment (varunsudhakar) [inline readme/the-dev-environment.md:17]: Can we state somewhere that an admin client is on its way to allow developers to create their own markets?

File: README.md (hunk 5 above)
Comment (druspencer) [reply README.md:56]: adding!

File: readme/the-roadmap.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+# Roadmap
+
+Here is what The Monaco Protocol Foundation is currently working on from both a technical and foundation point of view.
+
+# Monaco Protocol Foundation
```
Comment (varunsudhakar) [inline readme/the-roadmap.md:5]: I would remove the 'Publish whitepaper' and 'Form official foundation entity' - just have the technical writings below.

File: readme/the-dev-environment.md (hunk 6 above)
Comment (druspencer) [reply readme/the-dev-environment.md:17]: oh yeah, I can't believe I made that _less clear_

File: readme/the-roadmap.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
...
+
+- Publish whitepaper
+- Form official foundation entity 
+
+# Technical
```
Comment (varunsudhakar) [inline readme/the-roadmap.md:10]: Complete **security** audit of the protocol. 
***Further** open source the monaco protocol
Allow code contributions to the SDK and protocol

File: readme/the-roadmap.md (hunk 7 above)
Comment (druspencer) [reply readme/the-roadmap.md:5]: ✅

File: README.md (hunk 1 above)
Comment (druspencer) [reply README.md:19]: can probably stick with it for now though then change it every week :P

File: README.md [hunk 9]
```diff
@@ -13,3 +13,47 @@ Once you've done that, check out:
...
+{% hint style="danger" %} Please be vigilant when interacting with any program claiming to be the Monaco Protocol. Program addresses and authorities will always be shared by the Monaco Protocol Foundation and publicized through official channels such as these docs.{% endhint %}
+
+## How do I contribute to the Protocol?
+
+- More information about this will be available soon.
+- For now you can share ideas over on the [DevHub](https://github.com/MonacoProtocol/sdk/discussions).
+
+## Have any applications built on the Monaco Protocol?
+
+- BetDEX exchange is currently in closed beta.
```
Comment (varunsudhakar) [inline README.md:49]: Can we change to 'The BetDEX Exchange'?

**Review by stuartbetdex — APPROVED** [review]

**Pull request merged**